### ISSUE TITLE
does not work anymore?

### ISSUE THREAD

**Issue by nocturn50** [igdmapps/igdm#1394]
I used IGDM for quiet a while, very great program! But it does not work anymore :-( When I start up, I only get a "blank" screen
Please help me to get the chat function back, it's such I good program to use! 
Thanks in advance for your help!
![IGDM](https://user-images.githubusercontent.com/83336211/116472173-d6019500-a875-11eb-9894-78ef9e77614d.JPG)

**Comment by gabster1234** [comment]
yeah the same thing happens to me but I'm sorry i don't have a solution
